src/sentry/services/hybrid_cloud/user/__init__.py

```diff
@@ -196,6 +252,22 @@ def serialize_user(cls, user: User) -> APIUser:
             roles = frozenset(flatten(user.roles))
         args["roles"] = roles
 
+        options: FrozenSet[APIUserOption] = frozenset({})
+        if hasattr(user, "options") and user.options is not None:
+            options = frozenset(
```

Comment:
I think this is smart, and a good strategy.  But, I worry, as I mentioned above, about how it is opaque wether a user's options were loaded or not once you have a `APIUser` object.  I think, there is another strategy that I want to experiment with real quick.

Reply:
Changed my mind, this works.  Leaving as is.